tests/helpers/model.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, num_inputs=10, n_hidden=10, num_outputs=3):
         self.layer1 = torch.nn.Linear(num_inputs, n_hidden)
         self.dropout = torch.nn.Dropout(p=0.5)
         self.classifier = torch.nn.Linear(n_hidden, num_outputs)
-
+        self.encoder = torch.nn.Sequential(self.layer1, self.dropout)
```

Comment:
this encoder seems to be missing the activation function? the `features` method uses `tanh` after `layer1`.